packages/block-library/src/columns/index.js

```diff
@@ -79,6 +37,12 @@ export const settings = {
 			type: 'number',
 			default: 2,
 		},
+		verticalAlignment: {
+			type: 'string',
+		},
+		childColumnHasVerticalAlignmentOveride: {
```

Comment:
`childColumnHasVerticalAlignmentOveride` is never used in other places in this PR or the `master` branch. Should it be removed?

Reply:
Great spot. I introduced this during an experiment and forgot to remove. I have now got rid of it.